### ISSUE TITLE
SC 1.3.4 is problematic for some content appearing only on stationary hardware

### ISSUE THREAD

**Issue by mitchellevan** [w3c/wcag2ict#787]
### Issue

WCAG2ICT over-stretches the "essential" exception, to give a pass to software that runs only on devices such as TVs or kiosks. I agree it makes sense to give such software a pass for SC 1.3.4, but I believe the normative meaning of the "essential" exception is not quite up to this task.

As @GreggVan wrote in [EN 301 549 GitLab issue 660, comment 2025-09-30](https://labs.etsi.org/rep/HF/en301549/-/issues/660#note_27298):

> An essential exemption means that if you did this requirement - you product would no longer work.   That is is ESSENTIAL that this requirement NOT be followed. In this case -- making software reorient would not be a problem.  Just useless extra work - for no benefit.

Consider these use cases:

1. Content is designed for a projector or TV, but also appears on other devices.
2. Software is designed for the specific hardware of a handheld calculator, with closed functionality and no support for orientation changes.
3. A kiosk app is designed to be run on a platform capable of orientation changes, such as an Android tablet, but in practice the integrated kiosk will only be deployed with a single orientation in a fixed mounting.

I can accept applying the essential exception to use case \#2, because rotating the calculator software would make it not understandable in relation to the non-rotatable physical keypad.

It's in use case \#3 that the "essential" exception becomes most over-stretched. We're currently saying this: _if the restriction to a specific display orientation were removed, it would fundamentally change the information or functionality of the content_. But that's not true in use case \#3. In the unlikely event that somebody did physically rotate the kiosk, allowing the kiosk app to change orientation would have no negative impact on its function.

### Quoting our current texts

WCAG [Success Criterion 1.3.4 Orientation](https://www.w3.org/TR/WCAG22/#orientation):

> Content does not restrict its view and operation to a single display orientation, such as portrait or landscape, unless a specific display orientation is [essential](https://www.w3.org/TR/WCAG22/#dfn-essential).
>
> Note
Examples where a particular display orientation may be essential are a bank check, a piano application, slides for a projector or television, or virtual reality content where content is not necessarily restricted to landscape or portrait display orientation.

WCAG [definition of 'essential'](https://www.w3.org/TR/WCAG22/#dfn-essential):

> if removed, would fundamentally change the information or functionality of the content, **and** information and functionality cannot be achieved in another way that would conform

WCAG [Understanding SC 1.3.4](https://www.w3.org/WAI/WCAG22/Understanding/orientation.html):

> The exception for things considered essential is aimed at situations where the content would only be understood in a particular orientation, or where the technology restricts the possible orientations. If content is aimed at a specific environment which is only available in one orientation (such as a television) then the content can restrict the orientation. Technologies such as virtual reality use screens within goggles that cannot change orientation relative to the user's eyes.

WCAG2ICT, [Applying SC 1.3.4 Orientation to Non-Web Documents and Software](https://www.w3.org/TR/wcag2ict/#applying-sc-1-3-4-orientation-to-non-web-documents-and-software):

> This applies directly as written, and as described in [Intent from Understanding Success Criterion 1.3.4](https://www.w3.org/WAI/WCAG22/Understanding/orientation#intent).
> 
> Note 1 (Added)
> Content that is only used on hardware with a fixed display orientation or that has no sensor to detect or change the orientation is covered under the essential exception and does not need to provide support for orientation changes.
> 
> Note 2 (Added) (for non-web software)
> See also the [Comments on Closed Functionality](https://www.w3.org/TR/wcag2ict/#comments-on-closed-functionality).

WCAG2ICT, [Success Criteria Problematic for Closed Functionality](https://www.w3.org/TR/wcag2ict/#success-criteria-problematic-for-closed-functionality):

> [1.3.4 Orientation](https://www.w3.org/TR/wcag2ict/#orientation) — ICT with closed functionality that has fixed-in-place displays or other limitations that prevent modifying the physical display orientation should be considered as examples that are covered under the essential exception. See the note in the section [Applying SC 1.3.4 Orientation to Non-Web Documents and Software](https://www.w3.org/TR/wcag2ict/#applying-sc-1-3-4-orientation-to-non-web-documents-and-software).

### Proposed

Update the WCAG2ICT "problematic" section to address use case \#3 above. The update could be similar to Gregg's proposal in [EN 301 549 GitLab issue 660, comment 2025-09-29](https://labs.etsi.org/rep/HF/en301549/-/issues/660#note_27134).

Consider also editing the WCAG Understanding.

**Comment by maryjom** [comment]
We have been working on the appropriate language for the EN first and then will decide what to do with this. PR #780 is where we're working on the changes.

**Comment by maryjom** [comment]
PR 780 has been updated with the latest text from the EN 301 549 V20 draft. The TF discussed it today as well, though we didn't have a quorum to approve the PR.
